scripts/generate/templates/SUBSCRIPTION_MANAGEMENT/vanilla.ts.template

```diff
@@ -1,75 +1,234 @@
 import {
   ExtensionPoint,
+  ExtensionPointCallback,
   render,
   TextField,
-  ExtensionPointCallback,
+  Text,
+  Stack,
+  Checkbox,
 } from '@shopify/argo';
 
 const Add: ExtensionPointCallback[ExtensionPoint.SubscriptionManagementAdd] = (
   root,
   api
 ) => {
-  const textElement = root.createComponent(TextField, {
-    value: 'From my app: Add Action',
-    onAfterChange(value) {
-      textElement.updateProps({
-        value,
-      });
-    },
+  let selectedPlans: string[] = [];
+  const data = api.data;
+  const {primaryAction, secondaryAction, closeModal} = api.modalActions;
+  const mockPlans = [
+    {name: 'Subscription Plan A', id: 'a'},
+    {name: 'Subscription Plan B', id: 'b'},
+    {name: 'Subscription Plan C', id: 'c'},
+  ];
+
+  primaryAction.setContent('Add to plan');
+  primaryAction.setAction(() => {
+    data.done();
+    closeModal();
+  });
+
+  secondaryAction.setContent('Cancel');
+  secondaryAction.setAction(() => {
+    closeModal();
+  });
+
+  const textElement = root.createComponent(Text);
+  textElement.appendChild(
+    root.createText(
+      `Add {Product id ${data.productId}} to an existing plan or existing plans`
+    )
+  );
+  root.appendChild(textElement);
+
+  const stack = root.createComponent(Stack);
+
+  mockPlans.forEach((plan) => {
+    const checkbox = root.createComponent(Checkbox, {
+      label: plan.name,
+      checked: selectedPlans.includes(plan.id),
+      onChange: (checked) => {
+        selectedPlans = checked
+          ? selectedPlans.concat(plan.id)
+          : selectedPlans.filter((id) => id !== plan.id);
+
+        checkbox.updateProps({
+          checked: selectedPlans.includes(plan.id),
+        });
+      },
+    });
+
+    stack.appendChild(checkbox);
```

Comment:
TS shows an error for this and a bunch of other `appendChild` calls for some reason

![image](https://user-images.githubusercontent.com/29458473/84969944-b80cc500-b0e7-11ea-9083-6ce8450e013e.png)

Reply:
I need @henrytao-me 's PR for it to be fixed.

https://github.com/Shopify/app-extension-libs/pull/755#discussion_r441795536